### PR TITLE
Account management sub section creation

### PR DESCRIPTION
**Pull request by l0k0ms**
This is only for the english version of the doc.

All content in `/guides/saml` and `/guides/multiaccount` have been moved into the /account_management sub section. 

Content is pretty much the same, only minor updates.

This PR is up to date with :
* Master
* Chef guide merging: #1688
* Azure guide merging : #1689
* Infrastructure merging: #1717
* Monitor merging: https://github.com/DataDog/documentation/pull/1719

Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/account-management/account_management/

### CODE REVIEW

File: content/account_management/faq.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+autotocdepth: 2
+customnav: accountmanagementnav
+---
+
+### How do I setup my team in Datadog?
+
+The admin of the account should enter the email addresses of team members
+[here][other-1]. Some team best practices are as follows:
+
+* When the team member receives the confirmation email, they will be provided with a link to log in directly. The user should not click ‘sign up’ during this process.
```
Comment (irabinovitch) [inline content/account_management/faq.md]: Is this last sentence required "The user should not click ‘sign up’ during this process." ?

File: content/account_management/faq.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* If multiple users from the same organization sign up separately, this will register as different organizations in Datadog. Please reach out to support to have these merged, but please note that all information contained in the account getting merged will not be transferred over.
```
Comment (irabinovitch) [inline content/account_management/faq.md]: If multiple users from the same organization sign up separately, it will create multiple organizations which do not share data. If your team has created multiple organizations and would like to merge them, support can assist in doing so.  However, please note that only users are transferred. Data such as metrics or monitors created in the account being merged will not be transferred.

File: content/account_management/faq.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* The only access controls we have right now are around admin activities (adding/removing users, billing, etc.). As far as data goes (hosts, metrics, dashboards, etc.) all users have access to everything; more robust access controls are in our pipeline, but not something we’ve focused a lot of attention on yet.
```
Comment (irabinovitch) [inline content/account_management/faq.md]: Users can be placed in one of three permissions categories which include Read Only, Standard, and Administrator, Standard User and Read Only.   All three categories are able to view metrics, monitors, dashboards and other account data.  

Read Only users are able to all data, excluding billing information, but will be unable to make changes to dashboards, monitors, or integrations.

Standard users have read and write access to all Datadog functions such as installation or configuration of integrations, dashboards, and monitor creation. Standard users do not have access to user management or billing workflows.

Administrators have access to all read only and standard user features, but may also manage/invite users in the Datadog organization, change subscription plans, and view billing reports.

File: content/account_management/faq.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+* To remove a team member use the “disable” button on the same ‘team’ page (only available for admins). You cannot permanently remove users, just disable; disabled users will only be visible to admins on the team page and can’t log in and any session they have open is invalidated. We don’t fully delete them because they might own events, dashboards, etc. which are not supposed to be removed.
```
Comment (irabinovitch) [inline content/account_management/faq.md]: To disable a team member's access to an organization, an admin user must click the “disable” button by a given user's avatar on the [team](https://app.datadoghq.com/account/team) page. Disabled users will remain visible on the team page, but in the disabled state.  Disabled users/team members will disappear from the admin's team page UI automatically after one month.

File: content/account_management/faq.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+
+### Are my data and credentials safe?
+
+* Traffic is always initiated by the agent to Datadog. No sessions are ever initiated from Datadog back to the agent.
+* All traffic is sent over SSL.
+* All communication to Datadog is via HTTPS.
```
Comment (irabinovitch) [inline content/account_management/faq.md]: Are both points needed? SSL and HTTPS?

File: content/account_management/faq.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+* The full license agreement can be found [here][other-2].
+* The agent is entirely open source and can be found [here][other-3].
```
Comment (irabinovitch) [inline content/account_management/faq.md]: The Datadog Agent source code is available [here][other-3] under an open source software license.

File: content/account_management/faq.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+* Some installations (for example, installing the agent on CentOS 5), will request your password. The password is needed because it's installing packages - Datadog does not retain it in anyway. You can also use the step-by-step directions if you prefer to see exactly what the script is doing.
```
Comment (irabinovitch) [inline content/account_management/faq.md]: The installation process for the Datadog Agent and other components may prompt you for your administrative or root credentials.  The password is only used to complete the installation process, Datadog does not retain these credentials.  If you prefer to see the installation process step-by-step instructions can be found on the [agent installation]( https://app.datadoghq.com/account/settings#agent) page.

File: content/account_management/index.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
+---
+title: Account Management 
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+alias: 
+    - /guides/billing
+---
```
Comment (irabinovitch) [inline content/account_management/index.md]: @l0k0ms Please discuss with @marcweisman . He  has requested pricing not be present in the docs site. It should remain in the KB or link to the pricing page.

File: content/account_management/index.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+## Plan Overview
+
+There are three plans within Datadog:
+
+
+1. **Free** is only for 5 hosts or less. Free has a single day of data retention. The
+    good news is that switching from the Free plan to Pro will not affect your
+    setup, so anything you've begun monitoring would not be affected by that status
+    change.
+1. **Pro** is for 6-999 hosts and comes with 13 months of data retention.
```
Comment (irabinovitch) [inline content/account_management/index.md]: Where did this info come from? We have 15 months retention for metrics.

File: content/account_management/index.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+and 100 custom metrics)
+    1. Size of your environment in servers
+    1. Payment terms (month to month or annual prepaid)
+
+
+Custom metrics are supported in every plan. A custom metric is any metric that is not automatically collected by any of [Datadog’s integrations](https://www.datadoghq.com/product/integrations/)—for example custom checks or API-level metrics from your application. Each host may submit up to 100 custom metrics at no additional cost.
+
+Docker Containers are also supported in every plan. Each host may submit metrics from 10 containers an hour at no additional cost. Additional containers will be billed at $0.002 per container per hour. In addition, Enterprise customers can purchase prepaid containers at $1 per container per month.
+
+Pro and Enterprise data retention is for 13 months at full resolution (maximum is one point per second). For greater data retention needs, please reach out to [sales@datadoghq.com](mailto:sales@datadoghq.com).
```
Comment (irabinovitch) [inline content/account_management/index.md]: 15 months rather than 13.

File: content/account_management/index.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+    offered in the Pro plan. Enterprise includes phone support and pricing is based on three factors:
+    1. Data retention requirements and number of custom metrics (base plan includes 13 months retention,
+and 100 custom metrics)
+    1. Size of your environment in servers
+    1. Payment terms (month to month or annual prepaid)
+
+
+Custom metrics are supported in every plan. A custom metric is any metric that is not automatically collected by any of [Datadog’s integrations](https://www.datadoghq.com/product/integrations/)—for example custom checks or API-level metrics from your application. Each host may submit up to 100 custom metrics at no additional cost.
+
+Docker Containers are also supported in every plan. Each host may submit metrics from 10 containers an hour at no additional cost. Additional containers will be billed at $0.002 per container per hour. In addition, Enterprise customers can purchase prepaid containers at $1 per container per month.
```
Comment (irabinovitch) [inline content/account_management/index.md]: This is incorrect. Enterprise have access to 20 containers.

File: content/account_management/index.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+* [How do I see what I’ll get charged for this current month?](https://help.datadoghq.com/hc/en-us/articles/216095263-How-do-I-see-what-I-ll-get-charged-for-this-current-month-)
+
+* [Do non-reporting or inactive hosts count towards my bill?](https://help.datadoghq.com/hc/en-us/articles/203037549-Do-non-reporting-or-inactive-hosts-count-towards-my-bill-)
+
+* [What are Kubernetes pause containers and will they impact my bill?](https://help.datadoghq.com/hc/en-us/articles/115002102206-What-are-Kubernetes-pause-containers-and-will-they-impact-my-bill-)
+
+* [How can I change the Billing contact?](https://help.datadoghq.com/hc/en-us/articles/203764785-How-can-I-change-the-Billing-contact-)
+
+* [I don't see anything in my billing history](https://help.datadoghq.com/hc/en-us/articles/203844125-I-don-t-see-anything-in-my-billing-history)
+
```
Comment (irabinovitch) [inline content/account_management/index.md]: Lets delete this whole page and link to the KB.

File: content/account_management/saml.md [hunk 13]
```diff
@@ -97,6 +98,4 @@ The normal workflow is that when the Datadog url is loaded, the browser is redir
 
 In the normal setup, we won't know which org the assertion came from and this will result in an error page with a message saying that SAML Response is missing "InResponseTo" attribute.
 
-After enabling the feature (and waiting for caches to clear) the customer will need to get a new version of the SP Metadata, which will have a different, org-specific AssertionConsumerService endpoint to send assertions to.
-
-
+After enabling the feature (and waiting for caches to clear) the customer will need to get a new version of the SP Metadata, which will have a different, org-specific AssertionConsumerService endpoint to send assertions to.
```
Comment (irabinovitch) [inline content/account_management/saml.md]: @marcweisman  this look ok?

File: content/account_management/settings.md [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+---
+
+{{< img src="account_management/settings/settings_preferences.png" alt="Settings Preference" >}}
+
+This panel allow you: 
+
+* To chose your timezone
```
Comment (irabinovitch) [inline content/account_management/settings.md]: choose

File: content/account_management/settings.md [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+* Enable Desktop notifications
+* Custom your Email subscription parameters.
```
Comment (irabinovitch) [inline content/account_management/settings.md]: "Customize your email subscription settings"

Why does only one line have a period? We should be consistent.

File: content/account_management/team.md [hunk 16]
```diff
@@ -0,0 +1,29 @@
...
+title: Team
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+---
+
+## Add New Members
+
+1. To add members to a team, start by visiting the [Team Page](https://app.datadoghq.com/account/team).
+2. Enter the email address of the person you want to add to your team. Click **Invite Users**
```
Comment (irabinovitch) [inline content/account_management/team.md]: Enter the email address of the user you wish to invite to your Datadog account.

File: content/_index.md [hunk 17]
```diff
@@ -16,12 +16,14 @@ sidebar:
       href: "infrastructure/"
     - text: Integrations
       href: "integrations/"
+    - text: Monitor
+      href: "monitors/"
```
Comment (irabinovitch) [inline content/_index.md]: This looks like an unrelated change.

File: content/_index.ja.md [hunk 18]
```diff
@@ -46,7 +46,7 @@ aliases:
 <h3 class="big_number alert alert-success linked-header" id="dive-into-metrics2">
 <a class="header-link" href="#dive-into-metrics2"><span class="glyphicon glyphicon-link" aria-hidden="true"></span></a>2. メトリクスを <a href="guides/metrics/">使いこなそう </a>!</h3>
 <h3 class="big_number alert alert-info linked-header" id="graph-them3">
-<a class="header-link" href="#graph-them3"><span class="glyphicon glyphicon-link" aria-hidden="true"></span></a>3. <a href="graphing/">メトリクスの可視化</a>、そして<a href="guides/monitors/">アラート設定 !</a>
+<a class="header-link" href="#graph-them3"><span class="glyphicon glyphicon-link" aria-hidden="true"></span></a>3. <a href="graphing/">メトリクスの可視化</a>、そして<a href="https://docs.datadoghq.com/ja/guides/monitors/">アラート設定 !</a>
```
Comment (irabinovitch) [inline content/_index.ja.md]: This looks like an unrelated change.

File: content/account_management/faq.md (hunk 1 above)
Comment (l0k0ms) [reply content/account_management/faq.md]: It's a useful precision

File: content/account_management/faq.md (hunk 2 above)
Comment (l0k0ms) [reply content/account_management/faq.md]: 👍

File: content/account_management/faq.md (hunk 3 above)
Comment (l0k0ms) [reply content/account_management/faq.md]: 👍

File: content/account_management/faq.md (hunk 4 above)
Comment (l0k0ms) [reply content/account_management/faq.md]: 👍

File: content/_index.ja.md (hunk 18 above)
Comment (l0k0ms) [reply content/_index.ja.md]: 👍

File: content/_index.md (hunk 17 above)
Comment (l0k0ms) [reply content/_index.md]: 👍

File: content/account_management/faq.md (hunk 5 above)
Comment (l0k0ms) [reply content/account_management/faq.md]: yup. Our API work only in HTTPs I think

File: content/account_management/faq.md (hunk 6 above)
Comment (l0k0ms) [reply content/account_management/faq.md]: 👍

File: content/account_management/faq.md (hunk 7 above)
Comment (l0k0ms) [reply content/account_management/faq.md]: 👍

File: content/account_management/index.md (hunk 8 above)
Comment (l0k0ms) [reply content/account_management/index.md]: dully noted.

File: content/account_management/index.md (hunk 12 above)
Comment (l0k0ms) [reply content/account_management/index.md]: 👍

File: content/account_management/settings.md (hunk 15 above)
Comment (l0k0ms) [reply content/account_management/settings.md]: 👍

File: content/account_management/settings.md (hunk 14 above)
Comment (l0k0ms) [reply content/account_management/settings.md]: 👍

File: content/account_management/team.md (hunk 16 above)
Comment (l0k0ms) [reply content/account_management/team.md]: 👍
